lib/mix/lib/mix/state.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Mix.State do
```

Comment:
I don't care about this being ETS based. Mix is rarely used in a multi process setup. What do you think?